src/System.Runtime/samples/DateTime-formatting/DateTime-formatting.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Globalization;
+
+public class Program
+{
+	public static void Main()
+	{
+		// Demonstrates common uses of formatting the DateTime struct.
+		// Doc: https://msdn.microsoft.com/library/az4se3k1.aspx
+		
+		var date = DateTime.Now;
+		
+		// default formatting
+		Console.WriteLine("Print DateTime with default formatting: {0}", date);
+		Console.WriteLine();
+		
+		// short / long formatting
+		Console.WriteLine("Print short date: {0:d}", date);
+		Console.WriteLine("Print long date: {0:D}", date);
+		Console.WriteLine("Print short time: {0:t}", date);
+		Console.WriteLine("Print long time: {0:T}", date);
+		Console.WriteLine();
```

Comment:
Just as below the culture name is included in the output, it'd be good here to include what qualifier is being used, e.g.
```C#
Console.WriteLine("Print short date ('d'): {0:d}", date);
Console.WriteLine("Print long date ('D'): {0:D}", date);
Console.WriteLine("Print short time ('t'): {0:t}", date);
Console.WriteLine("Print long time ('T'): {0:T}", date);
```